Detector/DetDriftChamber/src/driftchamber/DriftChamber.cpp

```diff
@@ -237,34 +204,24 @@ static dd4hep::Ref_t create_detector(dd4hep::Detector& theDetector,
         }
   }
 
-  }
+//  }
 
     // - place in det
-    // inner
-    dd4hep::Transform3D transform_inner_chamber(dd4hep::Rotation3D(),
+    // - chamber
+    dd4hep::Transform3D transform_chamber(dd4hep::Rotation3D(),
             dd4hep::Position(0,0,0));
-    if(inner_chamber_enabled) {
-         dd4hep::PlacedVolume det_inner_chamber_phy = det_vol.placeVolume(det_inner_chamber_vol,
-                 transform_inner_chamber);
+    dd4hep::PlacedVolume det_chamber_phy = det_vol.placeVolume(det_chamber_vol,
+                 transform_chamber);
 
-         det_inner_chamber_phy.addPhysVolID("chamber", 0);
-    }
-    // outer
-    dd4hep::Transform3D transform_outer_chamber(dd4hep::Rotation3D(),
-            dd4hep::Position(0,0,0));
-    if(outer_chamber_enabled) {
-       dd4hep::PlacedVolume det_outer_chamber_phy = det_vol.placeVolume(det_outer_chamber_vol,
-               transform_inner_chamber);
+    det_chamber_phy.addPhysVolID("chamber", 0);
```

Comment:
For the chamber id, you should avoid the hardcoded number. 
Maybe the hardware groups need the multiple chambers in the future. 

Tao